Jenkinsfile.promote

```
@@ -159,7 +203,41 @@ void loginContainerRegistry(String registry, String credsId){
         sh "podman login -u ${REGISTRY_USER} -p ${REGISTRY_PWD} --tls-verify=false ${registry}"
     }
 }
+void readDeployProperties(){
+    if(params.DEPLOY_BUILD_URL != null){
+        sh "wget --auth-no-challenge --user=${JENKINS_USER} --password=${JENKINS_TOKEN} ${params.DEPLOY_BUILD_URL}artifact/deployment.properties"
```

Leftover from the PR of Kevin, we don't need the user and password

```suggestion
        sh "wget ${params.DEPLOY_BUILD_URL}artifact/deployment.properties"
```